algoliasearch-core/src/main/java/com/algolia/search/models/indexing/SearchResult.java

```diff
@@ -131,6 +133,15 @@ public String getQueryAfterRemoval() {
     return this;
   }
 
+  public RenderingContent getRenderingContent() {
+    return renderingContent;
+  }
+
+  public SearchResult<T> setRenderingContent(RenderingContent renderingContent) {
```

Comment:
I don't understand why this needs a public setter, but the other items seem to have it, so it's ok

Reply:
it is required by the serialization library (jackson)